### PR TITLE
Added instructions for adding executables to your PATH

### PR DESCRIPTION
**Pull request by hazmeister**

### CODE REVIEW

File: wd.html [hunk 1]
```diff
@@ -66,6 +66,19 @@
...
+<h4>On Windows</h4>
+<ul>
+    <li>Create a directory to place the executables in, like <em>C:\WebDriver\bin</em></li>
+    <li>Open a new cmd window as administrator and the run the following command to permanently add the directory to your path for all users on your machine:<pre><code>setx /m path "%path%;C:\WebDriver\bin\"</code></pre></li>
+    <li>Close all cmd windows and open a new one.  You can test by trying to run one of the binaries - e.g: <pre><code>chromedriver -v</code></pre>If that worked, you should be all set!</li>
+</ul>
+<h4>On OS X / Linux</h4>
+<ul>
+    <li>In a terminal: <pre><code>export PATH=$PATH:/path/to/dir >> ~/.profile</code></pre></li>
+</ul>
```
Comment (andreastt) [inline wd.html:99]: Use `<pre>`.

Also this only works when using the bash shell.

File: wd.html [hunk 2]
```diff
@@ -66,6 +66,19 @@
  This section explains the basic requirements
  for getting you started with the different browsers.
 
+<h3>Adding Executables to your PATH</h3>
+<p>Most drivers require an extra executable for Selenium to communicate with the browser. You can manually specify where the executable lives before starting WebDriver, but this can make your tests less portable, as the executables will need to be in the same place on every machine, or included within your test code repository.
+<p>By adding a folder containing WebDriver's binaries to your system's path, Selenium will be able to locate the additional binaries without requiring your test code to locate the exact location of the driver..
+<h4>On Windows</h4>
+<ul>
+    <li>Create a directory to place the executables in, like <em>C:\WebDriver\bin</em></li>
+    <li>Open a new cmd window as administrator and the run the following command to permanently add the directory to your path for all users on your machine:<pre><code>setx /m path "%path%;C:\WebDriver\bin\"</code></pre></li>
```
Comment (andreastt) [inline wd.html]: s/cmd window/command prompt/

Also drop the `<code>`.

File: wd.html [hunk 3]
```diff
@@ -66,6 +66,19 @@
...
+    <li>Close all cmd windows and open a new one.  You can test by trying to run one of the binaries - e.g: <pre><code>chromedriver -v</code></pre>If that worked, you should be all set!</li>
```
Comment (andreastt) [inline wd.html]: * Again, s/cmd windows/command prompts/.
* Drop `<code>`.
* The last sentence doesn’t make sense. Please fix grammar.

File: wd.html [hunk 4]
```diff
@@ -66,6 +66,19 @@
  This section explains the basic requirements
  for getting you started with the different browsers.
 
+<h3>Adding Executables to your PATH</h3>
+<p>Most drivers require an extra executable for Selenium to communicate with the browser. You can manually specify where the executable lives before starting WebDriver, but this can make your tests less portable, as the executables will need to be in the same place on every machine, or included within your test code repository.
+<p>By adding a folder containing WebDriver's binaries to your system's path, Selenium will be able to locate the additional binaries without requiring your test code to locate the exact location of the driver..
```
Comment (andreastt) [inline wd.html]: Double full stop.

**Review by andreastt — CHANGES_REQUESTED** [review]
Please also format so that no lines have more than ~72 characters per line.

File: wd.html (hunk 1 above)
Comment (hazmeister) [reply wd.html:99]: Is there a better option we could recommend which works for all shells?  Or could I get away with spelling out this way only works with bash?

**Comment by hazmeister** [comment]
Cheers for the review @andreastt - I've made some changes as requested.

**Comment by andreastt** [comment]
Thanks!

**Pull request merged**